tests/Behat/Context/GroupsContext.php

```diff
@@ -12,11 +12,24 @@
 
 use Behat\Behat\Context\Context;
 use Behat\Gherkin\Node\TableNode;
+use BitBag\SyliusProductAttributeGroupsPlugin\Entity\Attribute;
+use BitBag\SyliusProductAttributeGroupsPlugin\Entity\Group;
+use Doctrine\ORM\EntityManager;
 use Sylius\Behat\NotificationType;
+use Sylius\Behat\Page\Admin\Product\UpdateSimpleProductPage as BaseUpdateSimpleProductPage;
+use Sylius\Behat\Service\Helper\JavaScriptTestHelperInterface;
 use Sylius\Behat\Service\NotificationChecker;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Core\Formatter\StringInflector;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Repository\ProductRepositoryInterface;
+use Sylius\Component\Product\Model\ProductAttribute;
+use Sylius\Component\Product\Model\ProductAttributeTranslation;
 use Tests\BitBag\SyliusProductAttributeGroupsPlugin\Behat\Page\Admin\Group\CreateGroupPageInterface;
 use Tests\BitBag\SyliusProductAttributeGroupsPlugin\Behat\Page\Admin\Group\IndexGroupPageInterface;
+use Tests\BitBag\SyliusProductAttributeGroupsPlugin\Behat\Page\Admin\Product\UpdateSimpleProductPage;
 use Webmozart\Assert\Assert;
+use function PHPUnit\Framework\assertEquals;
 
 final class GroupsContext implements Context
```

Comment:
can u split Context between `setup, ui ..` etc ?

Reply:
referring to our earlier conversation - this will be corrected in opensource